programs/server/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ set (CLICKHOUSE_SERVER_LINK
 clickhouse_program_add(server)
 
 install(FILES config.xml users.xml DESTINATION ${CLICKHOUSE_ETC_DIR}/clickhouse-server COMPONENT clickhouse)
+install(FILES config.xml users.xml DESTINATION ${CLICKHOUSE_ETC_DIR}/clickhouse-server COMPONENT clickhouse)
+install(FILES config.d/query_masking_rules.xml DESTINATION ${CLICKHOUSE_ETC_DIR}/clickhouse-server/config.d COMPONENT clickhouse)
```

Comment:
Same as above. Probably it is better to add query_masking_rules directly to config.xml and don't touch config.d.